contracts/Removal.sol

```diff
@@ -945,4 +945,30 @@ contract Removal is
       revert InvalidData();
     }
   }
+
+  /**
+   * @notice Check if the amount and recipient constitute a valid transfer.
+   * @dev Ensure that the amount of tokens in circulation always multiples of 1e14.
+   *
+   * ##### Examples:
+   * - `_isValidTransfer({amount: 1e14, to: address(1)}) == true`
+   * - `_isValidTransfer({amount: 0, to: address(_certificate)}) == true`
+   * - `_isValidTransfer({amount: 1, to: address(1)}) == false`
+   * - `_isValidTransfer({amount: 1e14 - 1, to: address(_market)}) == false`
+   *
+   * ##### Requirements:
+   *
+   * - If the recipient is the Market or the Certificate, the amount must be divisible by 1e14 (100,000,000,000,000).
+   * - If the recipient is neither the Market nor the Certificate the amount may also be zero.
+   */
+  function _isValidTransfer(uint256 amount, address to)
+    internal
+    view
+    returns (bool)
+  {
+    return
```

Comment:
This ended up being a bit more complex than the original as it also accounts for 0 value removals